tests/core/resource/test_model.py

```diff
@@ -64,6 +64,13 @@ def test_delete_supports_collection_filters(self):
         records = result['data']
         self.assertEqual(len(records), 1)
 
+    def test_delete_collection_gives_number_of_deletable_records_in_headers(self):
+        self.resource.request.GET = {'_limit': '1'}
+        self.resource.collection_delete()
+        headers = self.last_response.headers
+        count = headers['Total-Records']
+        self.assertEquals(int(count), 2)
```

Comment:
I don't think this should go in the model tests, since it is done in the resource. `PaginatedDeleteTest` seems more appropriate